docs/conf.py

```diff
@@ -40,7 +48,24 @@
 # -- Options for HTML output ----------------------------------------------
 
 # We want to set the RTD theme, but not if we're on RTD.
-if os.environ.get('READTHEDOCS', '') != 'True':
+if os.environ.get('READTHEDOCS', '') == 'True':
```

Comment:
My case-sensitivity-bug spidey-sense is tingling. How important is it that values such as "TRUE", "true", "tru", and "tRuE" cause this expression to evaluate to False?